pkg/gadgets/bindsnoop/tracer/standard/tracer.go

```diff
@@ -40,7 +40,7 @@ func NewTracer(config *tracer.Config, resolver containercollection.ContainerReso
 		event.Type = eventtypes.NORMAL
 
 		if err := json.Unmarshal([]byte(line), &event); err != nil {
-			msg := fmt.Sprintf("failed to unmarshal event: %s", err)
+			msg := fmt.Sprintf("failed to unmarshal event: %s\n%s", err, line)
```

Comment:
nit: I am not sure if using this formatting is clear enough. For instance, when using JSON:

```JSON
{"type":"warn","message":"failed to unmarshal event: invalid character 'U' looking for beginning of value\nUnable to find kernel headers. Try rebuilding kernel with CONFIG_IKHEADERS=m (module) or installing the kernel development package for your running kernel version.","node":"vm-minikube-m02"}
```

What about this?
```suggestion
			msg := fmt.Sprintf("failed to unmarshal event '%s': %s", line, err)
```

The output would be:
```JSON
{"type":"warn","message":"failed to unmarshal event 'Unable to find kernel headers. Try rebuilding kernel with CONFIG_IKHEADERS=m (module) or installing the kernel development package for your running kernel version.': invalid character 'U' looking for beginning of value","node":"vm-minikube-m02"}
```

WDYT?